Microsoft.Toolkit.Uwp.UI.Controls/FlexPanel/FlexPanel.cs

```diff
@@ -0,0 +1,1572 @@
+            /// <value>The value for the property.</value>
+            /// <remarks>The default value for this property is NaN.</remarks>
+            public double Right { get; set; } = double.NaN;
+
+            /// <summary>Gets or sets  the shrink factor of the item.In case the child items overflow the main-axis of the container, this factor will be used to determine how individual items should shrink so that all items can fill inside the container.If this property is set to 0, the item will not shrink.</summary>
+            /// <value>The item shrink factor.</value>
+            /// <remarks>The default value for this property is 1 (all items will shrink equally).</remarks>
+            public double Shrink { get; set; } = ShrinkDefault;
+
+            /// <summary>Gets or sets  the top edge absolute position of the item. It also defines the item's height if <see cref="FlexItem.Bottom" /> is also set and if <see cref="FlexItem.Height" /> isn't set. It is ignored if <see cref="FlexItem.Position" /> isn't set to Absolute.</summary>
+            /// <value>The value for the property.</value>
+            /// <remarks>The default value for this property is NaN.</remarks>
+            public double Top { get; set; } = double.NaN;
+
+            /// <summary>Gets or sets  the width size dimension of the item.</summary>
+            /// <value>The width size dimension.</value>
+            /// <remarks>The default value for this property is NaN.</remarks>
+            public double Width { get; set; } = double.NaN;
+
+            /// <summary>Gets or sets  whether child items should be laid out in a single line(NoWrap) or multiple lines(Wrap or WrapReverse). If this property is set to Wrap or WrapReverse, <see cref = "FlexAlignContent" /> can then be used to specify how the lines should be distributed.</summary>
+            /// <value>Any value part of the<see cref="FlexWrap" /> enumeration.</value>
+            /// <remarks>The default value for this property is NoWrap.</remarks>
+            public FlexWrap Wrap { get; set; } = WrapDefault;
+
+            /// <summary>
+            /// Initializes a new instance of the <see cref="FlexItem"/> class.
+            /// </summary>
+            public FlexItem()
+            {
+            }
+
+            /// <summary>
+            /// Initializes a new instance of the <see cref="FlexItem"/> class.
+            /// </summary>
+            /// <param name="width">Width.</param>
+            /// <param name="height">Height.</param>
+            public FlexItem(double width, double height)
+            {
+                Width = width;
+                Height = height;
+            }
+
+            public void Add(FlexItem child)
+            {
+                Guard.IsNotNull(child, nameof(child));
+                Guard.IsNull(child.Parent, "child.Parent");
+                (children ?? (children = new List<FlexItem>())).Add(child);
```

Comment:
You can rewrite this with the compound assignment to make it less verbose 🙂

```suggestion
                (children ??= new List<FlexItem>()).Add(child);
```

Reply:
See below